modules/ROOT/pages/data_format.adoc

```diff
@@ -233,6 +233,40 @@ Use appropriate leading zeros for R and N to ensure proper ordering of files.
 
 **File structure**: Set of lines containing key: value.
 For anthropometric measures, the keys should be the ones presented in <<model.adoc#table_body_segment, body segment table>>.
+Each body segment is characterized by its name, mass, length, center of mass and inertia matrix.
```

Comment:
I would relax the sentence a bit, using _can be characterized_ instead of _is_ , so that we do not force all measurements to be provided.